src/simplewallet/simplewallet.cpp

```diff
@@ -5027,8 +5027,16 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
 
     if (amount < config::graft::TIER1_STAKE_AMOUNT)
     {
-      fail_msg_writer() << tr("locked amount ") << double(amount) / COIN << tr(" is less than minimum required ") << double(config::graft::TIER1_STAKE_AMOUNT) / COIN;
-      return true;
+      std::string accepted = command_line::input_line(tr("Stake amount ") + std::to_string(double(amount) / COIN) + tr(" is less than the minimum supernode stake ") +
```

Comment:
`input_line` is not in a `command_line` namespace anymore, could you please fix it?

Reply:
will do

Reply:
Fixed as requested